app/websocket/src/main/kotlin/com/xorker/draw/websocket/message/response/dto/MafiaGameInfoMessage.kt

```diff
@@ -0,0 +1,30 @@
+package com.xorker.draw.websocket.message.response.dto
+
+import com.xorker.draw.mafia.MafiaGameOption
+import com.xorker.draw.websocket.ResponseAction
+import com.xorker.draw.websocket.SessionMessage
+
+data class MafiaGameInfoMessage(
+    override val body: Any,
```

Comment:
h3. Body 타입 명시해도 될 것 같아요.

```suggestion
    override val body: MafiaGameInfoBody,
```